pom.xml

```diff
@@ -231,6 +231,21 @@
             <artifactId>swagger-annotations</artifactId>
             <version>2.2.2</version>
         </dependency>
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.gsonfire</groupId>
+            <artifactId>gson-fire</artifactId>
+            <version>1.8.5</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
```

Comment:
These changes will override the pom that is already on develop. that version is currently correct, so these changes can be dropped

Reply:
Done and also added a line in the makefile generation line to ignore reserved word mapping for configuration and ran the model generation again to update it.